misk/src/main/kotlin/misk/web/jetty/JettyWebSocket.kt

```diff
@@ -155,6 +158,7 @@ internal class JettyWebSocket(
 
       val httpCall = ServletHttpCall.create(
           request = request.httpServletRequest,
+          sourceSocket = SourceSocket(SourceSocket.Type.NETWORK, "websocket"),
```

Comment:
I don't know what name this should have, the request doesn't appear to have any (network) socket info, only request info. This means there isn't a way to get the host:port (other then parsing it out of the ws:// request URL)